src/site/layouts/page.drupal.liquid

```diff
@@ -0,0 +1,29 @@
+{% include "src/site/includes/header.html" %}
+<main>
+  <div class="row">
+    {% if !entityPublished %}
+      <div class="usa-alert usa-alert-info" >
+        <div class="usa-alert-body">
+          <p class="usa-alert-text">You are viewing a draft.</p>
+        </div>
+      </div>
+    {% endif %}
+
+    <article class="usa-content">
+      <h1>{{ title }}</h1>
+      <div class="usa-font-lead">
+        {{ fieldIntroText }}
+      </div>
+      {% for block in fieldContentBlock %}
+        {{ block.entity.entityRendered }}
```

Comment:
I'm assuming we'll either have a giant switch statement here (gross) or render each of these blocks using their specific templates before processing this template for non-wysiwyg content types.